docs/dev/openstack-howto.md

```diff
@@ -0,0 +1,93 @@
+# OpenStack HOWTO
+
+It's possible to launch an all-in-one - for test and development - cluster on OpenStack.
+
+## The ocp-doit scripts
+
+We've created a set of scripts that make it easier to setup an all-in-one OpenStack. The OpenStack cluster should work either on a VM or a BM node. We recommend the latter.
+
+The scripts can be found in the following [repo](https://github.com/imain/ocp-doit)
+
+
+### Configure ocp-doit
+
+Copy the `config_example.sh` file and configure it to match your environment.
+
+### Install OpenStack 
+
+Run the scripts in the following order to install the dependencies, deploy, and configure OpenStack:
+
```

Comment:
To be clear should we say "Run the scripts in the following order on the target host" ?